upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -239,6 +239,15 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		dest["EnableSQSTerminationDraining"] = func() bool { return *cluster.Spec.NodeTerminationHandler.EnableSQSTerminationDraining }
 	}
 
+	if cluster.Spec.CloudConfig != nil && fi.BoolValue(cluster.Spec.CloudConfig.ManageStorageClasses) {
```

Comment:
This sort of logic shouldn't be template functions. Template functions aren't scoped to any particular component or logic and makes things harder to maintain. That is precisely why we have option builders.

Reply:
If you mean that `fi.BoolValue(cluster.Spec.CloudConfig.ManageStorageClasses)` part should be removed, I agree.
If you mean that we should always materialize `AWSEBSCSIDriver.Enabled`, I am not so sure.

Reply:
We have to materialize it. Or make the templatest more complex.

The problem with the materialization now is that it triggers an unnecessary change to the bootstrap script. I think we can rather restrict what from `CloudConfig` is added to that.

Reply:
Think this was answered in office hours. Short version is that I wanted e2e to pass before enabling by default for all clusters. That work is almost done now.

Reply:
Agreed, answered and the PR was was updated.